server/bin/unittests

```diff
@@ -334,6 +334,10 @@ function _setup_state {
         exit $res
     fi
 
+    # Ensure proper expected umask.
+    find ${_testopt} ${_testopt_sat} -perm -u=rx -exec chmod 775 {} \;
+    find ${_testopt} ${_testopt_sat} -perm -u=r ! -perm -u=rx -exec chmod 664 {} \;
```

Comment:
> Why (in line 339) do you need to escape the `;` but not the `!`?

Because a `!` followed by a blank in the middle of a command is not subject to expansion or other meaning, apparently.

Reply:
The leading `-` in the `-u=*` arguments to `-perm` are important, see https://man7.org/linux/man-pages/man1/find.1.html.

Reply:
> The leading - in the -u=* arguments to -perm are important

Yes, they are...and, I'm specifically suggesting that we are better of _without_ them, see https://man7.org/linux/man-pages/man1/find.1.html.  😉

Reply:
If you can demonstrate that it works, I'll use it.

Reply:
I tried without the '-' and it did not work.

Reply:
I'll give it a try.  (What was the exact command line that you used which didn't work?)

Reply:
If you can make it work I'll be happy to take the approach that works.

Until this PR lands other CI jobs will be broken.

I have something in this PR that works, but don't have the time to apply all this changes to ensure they work and get this submitted.  I'll update what I can but after that I'll have to move on to other work.